client/components/text-diff/style.scss

```diff
@@ -0,0 +1,15 @@
+.text-diff__additions {
+	background: lighten($blue-wordpress, 58%);
+	border-bottom: 2px solid $blue-wordpress;
+}
+
+
+.text-diff__context {
+	// TODO: Should there be specific styles for this class? Should we just remove it altogther?
```

Comment:
The context class could be useful to apply styles (e.g. lightening the text color, collapsing & hiding..) to "copy" ops.

Maybe update this comment w/ some work that could happen there.

Reply:
👍 :)

Reply:
I've rebased the comment (and empty ref) out for now - I hadn't realised that context is really just 'anything not changed in this diff' until last night, so I now think it makes sense not to style it all, at least for now :)